app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+
+  def recent_users_post
+    posts.where(author_id.params[:id]).order(created_at: :desc).limit(3)
+  end
```

Comment:
### 🛑 Required Change 

- [ ] The posts returned should not be specific to an author/user. Kindly refer to theh suggestion below

```ruby
def recent_post
  posts.order('created_at Desc').limit(3)
end
```

Reply:
Thanks for your review, suggested changes implemented.